src/dev/precommit_hook/casing_check_config.js

```diff
@@ -102,14 +102,15 @@ export const KEBAB_CASE_DIRECTORY_GLOBS = [
   'packages/*',
   'x-pack',
   'x-pack/packages/*',
+  'src/core/packages/*/*',
```

Comment:
In fact, the core packages that were living under /packages/core don't have a special treatment (line 102), so perhaps we don't need the `/*/*`

```suggestion
 'src/core/packages/*',
```

Reply:
actually... I noticed that we'll have `src/core/packages/http/resources-server` so `/*/*` is necessary here.